index.js

```diff
@@ -33,6 +33,9 @@ class Parameter {
     if (opts.validateRoot) {
       this.validateRoot = true;
     }
+    this.defaultRequired = opts.defaultRequired !== undefined ?
+                        opts.defaultRequired :
```

Comment:
this.defaultRequired = opts.defaultRequired !== false;

Reply:
thx for guiding ! ~